service-api/module/Application/src/Services/NinoAPIService.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Services;
```

Comment:
Been thinking a bit more about this: I think all of these services should be in a separate namespace so that we have clear isolation. When we replace them with real integrations, things like authentication are going to be very different and will require support classes. Isolating them allows each to grow naturally.

So I'd suggest we end up with something like:
- `Application\Nino\Validator`
- `Application\Nino\ValidatorInterface`
- `Application\Nino\ValidatorFactory`

And then the mock should maybe live at:
- `Application\Mock\Validator`

Interested to hear what you think about that.

Reply:
Think it makes sense since we already have at 3/4 different types of these ID services itself